vertigo-bot-executor/src/main/java/io/vertigo/chatbot/engine/plugins/bt/command/bot/BotBtCommandParserDefinitionProvider.java

```diff
@@ -1,120 +1,121 @@
+				BtCommandParserDefinition.basicCommand("say", (c, p) -> BotNodeProvider.sayOncePerTree(getBB(p), c.getStringParam(0))),
+				BtCommandParserDefinition.basicCommand("say:always", (c, p) -> BotNodeProvider.say(getBB(p), c.getStringParam(0))),
+				BtCommandParserDefinition.basicCommand("say:once", (c, p) -> BotNodeProvider.sayOnce(getBB(p), c.getStringParam(0))),
+				BtCommandParserDefinition.basicCommand("eqInt", (c, p) -> BotNodeProvider.eq(getBB(p), c.getStringParam(0), c.getIntParam(1))),
+				BtCommandParserDefinition.basicCommand("eq", (c, p) -> BotNodeProvider.eq(getBB(p), c.getStringParam(0), c.getStringParam(1))),
+				BtCommandParserDefinition.basicCommand("gt", (c, p) -> BotNodeProvider.gt(getBB(p), c.getStringParam(0), c.getIntParam(1))),
+				BtCommandParserDefinition.basicCommand("lt", (c, p) -> BotNodeProvider.lt(getBB(p), c.getStringParam(0), c.getIntParam(1))),
+				BtCommandParserDefinition.basicCommand("topic", (c, p) -> BotNodeProvider.switchTopic(getBB(p), c.getStringParam(0))),
+				BtCommandParserDefinition.basicCommand("topic:start", (c, p) -> BotNodeProvider.switchTopicStart(getBB(p))),
+				BtCommandParserDefinition.basicCommand("topic:fallback", (c, p) -> BotNodeProvider.switchTopicFallback(getBB(p))),
+				BtCommandParserDefinition.basicCommand("topic:end", (c, p) -> BotNodeProvider.switchTopicEnd(getBB(p))),
+				BtCommandParserDefinition.compositeCommand("random", (c, p, l) -> BotNodeProvider.random(l)),
+				BtCommandParserDefinition.basicCommand("inputString", (c, p) -> {
+					final var optFirstChoice = c.getOptStringParam(2);
+					if (optFirstChoice.isEmpty()) {
+						return BotNodeProvider.inputString(getBB(p), c.getStringParam(0), c.getStringParam(1));
+					}
+					return BotNodeProvider.inputString(getBB(p), c.getStringParam(0), c.getStringParam(1), optFirstChoice.get(), c.getRemainingStringParam(3));
+				}),
+				BtCommandParserDefinition.basicCommand("fulfilled", (c, p) -> BotNodeProvider.fulfilled(getBB(p), c.getStringParam(0))),
+
+				BtCommandParserDefinition.compositeCommand("switch", BotBtCommandParserDefinitionProvider::buildSwitchNode),
+				BtCommandParserDefinition.compositeCommand("case", (c, p, l) -> new BotCase(c.getStringParam(0), l)),
+
+				BtCommandParserDefinition.basicCommand("choose:nlu", (c, p) -> BotNodeProvider.chooseNlu(getBB(p), c.getStringParam(0))),
+				BtCommandParserDefinition.compositeCommand("choose:button", BotBtCommandParserDefinitionProvider::buildChooseButtonNode),
+				BtCommandParserDefinition.compositeCommand("choose:button:nlu", BotBtCommandParserDefinitionProvider::buildChooseButtonNluNode),
+				BtCommandParserDefinition.basicCommand("button", (c, p) -> new BotButton(c.getStringParam(0), c.getStringParam(1))),
+				BtCommandParserDefinition.basicCommand("jsEvent:start", (c, p) -> BotNodeProvider.launchJsEvent(getBB(p), c.getStringParam(0))));

```

Comment:
```suggestion
				BtCommandParserDefinition.basicCommand("jsEvent", (c, p) -> BotNodeProvider.launchJsEvent(getBB(p), c.getStringParam(0))));
```
le `:start` me semble pas pertinent.